server/sonar-server/src/test/java/org/sonar/server/rule/ws/SearchActionTest.java

```diff
@@ -177,6 +177,67 @@ public void filter_by_rule_key() {
   }
 
   @Test
+  public void filter_by_rule_name() {
```

Comment:
I wonder if all these tests should be moved to RuleIndexTest because they aim to validate the ES request. WDYT ?

Reply:
Indeed we have (at least) two places for those test at the moment. In my opinion this also applies to all the other indexes (Component, Issues, etc.).

Since I have quite limited time today, I would love to not do this refactoring now.